packages/IMAPClient-Core.package/ICEndPoint.class/instance/testAccountWith..st

```diff
@@ -2,6 +2,5 @@ login
 testAccountWith: anAccountInfo
 
 	"self connectAndLoginWith: anAccountInfo.
- 
-	^ self fatalErrorOccured not"
-	^ true
\ No newline at end of file
+	^ self connectionHandler fatalErrorOccured not"
+	^true
```

Comment:
should probably not constantly return true in the merged version

Reply:
that is dead code, will be removed (source: jan)